firefly/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "58481132-c79e-4659-8064-7cdaabbbc999",
+  "app_id": "firefly",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Bring your cloud Up-to-Code",
+    "title": "Firefly",
+    "media": [
+      {
+        "caption": "Full Cloud Inventory",
```

Comment:
The images look a bit small and are pixelated when zooming in - would you be able to increase the resolution on these?